influxql/parser_test.go

```diff
@@ -1717,10 +1717,13 @@ func TestParser_ParseStatement(t *testing.T) {
 		{s: `DROP FOO`, err: `found FOO, expected SERIES, CONTINUOUS, MEASUREMENT, SERVER, SUBSCRIPTION at line 1, char 6`},
 		{s: `CREATE FOO`, err: `found FOO, expected CONTINUOUS, DATABASE, USER, RETENTION, SUBSCRIPTION at line 1, char 8`},
 		{s: `CREATE DATABASE`, err: `found EOF, expected identifier at line 1, char 17`},
+		{s: `CREATE DATABASE "testdb" "db2"`, err: `found db2, expected EOF at line 1, char 26`},
 		{s: `CREATE DATABASE "testdb" WITH`, err: `found EOF, expected DURATION, REPLICATION, NAME at line 1, char 31`},
 		{s: `CREATE DATABASE "testdb" WITH DURATION`, err: `found EOF, expected duration at line 1, char 40`},
 		{s: `CREATE DATABASE "testdb" WITH REPLICATION`, err: `found EOF, expected number at line 1, char 43`},
 		{s: `CREATE DATABASE "testdb" WITH NAME`, err: `found EOF, expected identifier at line 1, char 36`},
+		{s: `CREATE DATABASE "testdb" WITH DURATION 24h REPLICATION 1 "db2"`, err: `found db2, expected NAME at line 1, char 58`},
+		{s: `CREATE DATABASE "testdb" WITH DURATION 24h REPLICATION 1 NAME "testrpi" "db2"`, err: `found db2, expected EOF at line 1, char 73`},
```

Comment:
Thanks @pires -- quoting handles some edge cases in our code, and I want to make sure it doesn't confuse anything.